src/Store.php

```diff
@@ -33,6 +33,9 @@ public function getStock($product_id): int {
       $product_id = $product_id->get_id();
     }
     $value = get_post_meta($product_id, $this->getStockMetaKey(), TRUE);
+    if (empty($value)) {
+      $value = 0;
+    }
     return $value;
```

Comment:
You can use a ternary operator here:

```suggestion
    if (empty($value)) {
      $value = 0;
    }
    return empty($value) ? 0 : $value;
```

Or this might also work

```suggestion
    return $value ?? 0;
```

Not sure if this would work, but it might also be possible to combine line 35 into the return 

```suggestion
    return get_post_meta($product_id, $this->getStockMetaKey(), TRUE) ?? 0;
```